sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -775,6 +778,57 @@ def _get_sample_count(self, n_samples):
             return ceil(n_samples * self.subsample)
         return n_samples
 
+    def _get_color_dict_from_values(
+        self, value_list, categorical_or_continuous: str, cmap_name: str = "Spectral"
```

Comment:
statt `categorical_or_continuous` könntest du auch ein bool `is_categorical` nehmen

Reply:
bzw dadurch dass du das eh unten in der funktion aufrufst, wo du ja nochmal den check hast, was es jetzt ist, könntest du auch einfach 2 getrennte funktionen machen

Reply:
und da das hier unabhängig von der klasse ist, würde ich das auch nicht als klassenmethode machen, sondern außerhalb